CommunityToolkit.App.Shared/Renderers/ToolkitSampleRenderer.xaml.cs

```diff
@@ -193,10 +193,13 @@ private async Task LoadData()
             // Generated properties reference these in getters and setters.
             propertyContainer.GeneratedPropertyMetadata = Metadata.GeneratedSampleOptions;
 
-            SampleOptionsPaneInstance = new GeneratedSampleOptionsRenderer
+            if (propertyContainer.GeneratedPropertyMetadata is not null)
             {
-                SampleOptions = propertyContainer.GeneratedPropertyMetadata
-            };
+                SampleOptionsPaneInstance = new GeneratedSampleOptionsRenderer
+                {
+                    SampleOptions = propertyContainer.GeneratedPropertyMetadata.ToList()
```

Comment:
Should we just make this interface a list as well? Won't ToList() be more expensive here?

Reply:
This would require changes to the source generator. Digging through the code, it's generated as an array, so I'll use that instead.

Reply:
Fixed in 76501d0842ab3f3d86e3bdd035a9bb9b2b645515